lightbluetent/templates/base.html

```diff
@@ -17,6 +17,7 @@
   <link href="https://www.srcf.net/_srcf/vendor/css/font-awesome.min.css" rel="stylesheet">
   <link href="https://www.srcf.net/_srcf/srcf-bs.css" rel="stylesheet" type="text/css">
   <link href="{{ url_for('static', filename='styles.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ url_for('static', filename='red-pulse.css') }}" rel="stylesheet" type="text/css">
```

Comment:
Nice touch! But we can merge this in the styles.css file which we inherit from anyway